src/lib/isAlpha.js

```diff
@@ -1,8 +1,22 @@
 import assertString from './util/assertString';
 import { alpha } from './alpha';
 
-export default function isAlpha(str, locale = 'en-US') {
-  assertString(str);
+export default function isAlpha(_str, locale = 'en-US', options = {}) {
+  assertString(_str);
+
+  let str = _str;
+  const { ignore } = options;
```

Comment:
I am with the opinion that `ignore` characters should be optional. 
This actually fails if a user does not pass ignore values (It is not guaranteed that a user needs to have some values ignored). 
Also, if I pass options as  `args: [{ ignore: /[\s/-]/g }]`, it will fail or I do pass an `args: ['en-US', { }]` it also fails. 
I am open to discussion on way forward. 

Cc. @profnandaa

Reply:
hmm technically i don't see a possibility of failing for any of the above args, may be i can add those

https://github.com/validatorjs/validator.js/blob/a919d99cfda475d2cf23e28de0bae6627e32b61d/src/lib/isAlpha.js#L8-L10

the above line says that ignore is optional which means, `args: ['en-US', { }]` should pass

as far as about this `[{ ignore: /[\s/-]/g }]` we are already asserting first argument to be string


let me know if it makes sense @profnandaa @ezkemboi